src/TouristInfoCards.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const TouristInfoCards = () => (
```

Comment:
each of these cards have the same format so it would be good if either a) they were a component or b) use a `map` function to take the data (city name, image, link url etc) and return the card